src/xenia/kernel/xboxkrnl/xboxkrnl_video.cc

```diff
@@ -136,9 +143,9 @@ void VdQueryVideoMode(X_VIDEO_MODE* video_mode) {
   std::memset(video_mode, 0, sizeof(X_VIDEO_MODE));
   video_mode->display_width = 1280;
   video_mode->display_height = 720;
-  video_mode->is_interlaced = 0;
```

Comment:
Is it the case that all 3 of these fields support all possible combinations as seen in the below truth table?

| widescreen | hi_def | interlaced |
|---|---|---|
| 0 | 0 | 0 |
| 0 | 0 | 1 |
| 0 | 1 | 0 |
| 1 | 0 | 0 |
| 0 | 1 | 1 |
| 1 | 0 | 1 |
| 1 | 1 | 0 |
| 1 | 1 | 1 |

If there are combinations that aren't supported, I'd be in favor of having just a single string cvar flag with pre-defined known working combinations for all three fields

Reply:
Yes, all possible combinations of the `widescreen`, `hi_def`, and `interlaced` fields are supported. There are no unsupported combinations. 
I appreciate your suggestion! I will consider this for future updates.